gcp-js-webserver/Pulumi.yaml

```diff
@@ -8,3 +8,6 @@ template:
     gcp:zone:
       description: The Google Cloud zone
       default: us-central1-a
+    gcp:region:
+      description: The Google Cloud region to place the subnet in
+      default: us-central1
```

Comment:
Can we use only one or the other of these?

Also - unfortunately for this change - this code is used in https://www.pulumi.com/docs/tutorials/gcp/gce-webserver/, which points back to this - so we really want to keep these in sync - and also really want to keep this as absolutely simple as possible. 

It may be that because of that, it’s not actually the right thing to make any change here. How serious of a concern is it to reduce the subnet load of these tests?  Are there other ways we could mitigate that (limit increases?).